### ISSUE TITLE
CSS: broken comment/uncomment action

### ISSUE THREAD

**Issue by fabiospampinato** [Microsoft/vscode#30554]
- VSCode Version: 1.14
- OS Version: macOS 10.13 Beta 1

Steps to Reproduce (bug#1):

1. Open a new editor
2. Set the language to CSS
3. Create a comment (cmd+/)
4. The cursor doesn't get inserted in the middle (`/*|  */` instead of (`/* | */`)

Steps to Reproduce (bug#2):

1. Open a new editor
2. Set the language to CSS
3. Write `.foo {}`
4. Comment/uncomment the line (cmd+/) multiple times, extra spaces get added at the beginning of the line

Reproduces without extensions: Yes

**Comment by ZhangPuXi** [comment]
I also met this problem. When comment css code,  two spaces are inserted, but when I uncomment it, the spaces is still there. Maybe the spaces should be trimmed after uncomment.

**Comment by rebornix** [comment]
The second issue would be fixed by https://github.com/Microsoft/vscode/pull/30818

**Comment by cleidigh** [comment]
@rebornix 
once #30818  is done and okay
I could also looking to book number one since I'm already messing with this code
but I'd rather do it as a separate PR.

**Comment by rebornix** [comment]
@cleidigh go ahead and let me know if you need any tips. The change should not be too complex (changing the result selection).

**Comment by cleidigh** [comment]
@rebornix 
See PR #30964
